addons/main/initSettings.sqf

```diff
@@ -207,6 +207,28 @@ _category = [_header, LLSTRING(subCategoryFeedback)];
     false
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(visibleBleedoutTimer),
+    "LIST",
+    [LLSTRING(visibleBleedoutTimer), LLSTRING(visibleBleedoutTimer_desc)],
+    _category,
+    [[0, 1, 2], ["str_a3_cfgvehicles_modulesector_f_arguments_taskowner_values_nobody_0", "str_a3_rscdisplaycampaignlobby_mediconly", "str_a3_cfgvehicles_modulesector_f_arguments_taskowner_values_everyone_0"], 0],
```

Comment:
Should default be for medics only instead of none? Might be a better default setting?

Reply:
I set it to none b/c that's the current functionality.

Reply:
It is a new functionality not a change to an existing one, it is fine to have it on by default.